src/dex-ui/services/contractsUAT.json

```diff
@@ -204,13 +195,31 @@
     "transparentProxyAddress": "0x0000000000000000000000000000000000e4cc7b",
     "transparentProxyId": "0.0.14994555"
   },
+  {
+    "name": "ftdao",
+    "id": "0.0.15021369",
+    "address": "0x0000000000000000000000000000000000e53539",
+    "timestamp": "2023-06-30T10:42:41.565Z",
+    "hash": "0b822af7e01b4fd0992bd9776202eb0d",
+    "transparentProxyAddress": "0x0000000000000000000000000000000000e5353e",
+    "transparentProxyId": "0.0.15021374"
+  },
   {
     "name": "ftdaofactory",
-    "id": "0.0.14994650",
-    "address": "0x0000000000000000000000000000000000e4ccda",
-    "timestamp": "2023-06-27T09:01:43.658Z",
-    "hash": "d5f28caa0983238dd51ad7032bb4ce46",
-    "transparentProxyAddress": "0x0000000000000000000000000000000000e4ccdc",
-    "transparentProxyId": "0.0.14994652"
+    "id": "0.0.15045797",
+    "address": "0x0000000000000000000000000000000000e594a5",
+    "timestamp": "2023-07-04T12:29:21.865Z",
+    "hash": "e45de6ad8062b355fe44d00aa9061f07",
+    "transparentProxyAddress": "0x0000000000000000000000000000000000e594a7",
+    "transparentProxyId": "0.0.15045799"
+  },
+  {
+    "name": "nftdaofactory",
+    "id": "0.0.15021409",
+    "address": "0x0000000000000000000000000000000000e53561",
+    "timestamp": "2023-06-30T10:53:46.503Z",
+    "hash": "91b53d6e773b56141cef7ebd77d8853d",
+    "transparentProxyAddress": "0x0000000000000000000000000000000000e53565",
+    "transparentProxyId": "0.0.15021413"
   }
 ]
```

Comment:
1. 
<img width="406" alt="Screenshot 2023-07-05 at 3 03 23 PM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/89677167/c99087b9-e0d1-4876-8a2b-da6bce7a0112">

Quorum is not visible in UI.

2. 
<img width="400" alt="Screenshot 2023-07-05 at 3 00 43 PM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/89677167/4ff74137-528c-4983-bdfb-27669b347a3d">

I am not able to set Quorum as 0.1% of total supply minimum is 1%.

3.
<img width="630" alt="Screenshot 2023-07-05 at 2 57 35 PM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/89677167/1169c72d-9314-4ef1-a3d1-42ef0b4bb2f8">

Created By is missing in UI.

4. 
<img width="1324" alt="Screenshot 2023-07-05 at 2 57 19 PM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/89677167/b298beee-277b-4fb8-8518-47bb70f8fae8">
Under Details section we can remove Title as it coming in Top (As per Figma)

Reply:
1. Added Quorum in the Horizontal chart, and used the same horizontal chart we are using in Governance.
2. Quorum is incremented in 1/100th of the percentage. added logic to convert value while creating DAO.
3. Added the created by field.
4. In details it is not title, it is proposal type. @damiano-melcarne-lab49 created it. so I used it as it is.  Let me know if we want to remove it. This is the only place where proposal type is visible in UI.

Reply:
<img width="345" alt="Screenshot 2023-07-06 at 4 00 22 PM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/132666177/2c1856dc-daf2-4a16-b33e-c63b97fc29e7">
<img width="992" alt="Screenshot 2023-07-06 at 4 01 57 PM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/132666177/33b888fc-8eb2-43f0-a383-7a2517b2fe4b">
<img width="1064" alt="Screenshot 2023-07-06 at 4 02 52 PM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/132666177/255481fd-d33f-44aa-93e5-3927975ae6c6">

Reply:
@anshulkhare-lab49 
- Leave the proposal type for now. We need to holistically rethink the details section design. We can fix this in a future task.
- Convert the "created" address to a Hedera Account Id (`AccountId.fromSolidityAddress(created).toString()`). You can then use the `HashScanLink` component to convert the account id text into a clickable link.

Reply:
Removed the transaction field as in Gov DAO we don't have transaction hash.
Removed the created field as author is already shown in the the top.
Changed Style for Type. 
<img width="1536" alt="Screenshot 2023-07-07 at 4 40 43 PM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/132666177/5c88ad3a-e07b-435a-b454-06c1d25d93e4">